roles/cloudman-boot/tasks/main.yaml

```diff
@@ -13,6 +13,9 @@
 - name: Setup cert-manager
   include_tasks: certmanager.yaml
 
+- name: Setup Galaxy-CVMFS-CSI
```

Comment:
I was wondering, did you want to move this another line higher? cert-manager is not needed for cvmfs-csi right? So this buys another tiny bit of time for the preloading.

Reply:
Also, is the cache size no longer needed?
```
                  values:
                    cache:
                      size: 10000
```

Reply:
It would be cache.alienCache.size now. It can be set, but the default is 10G anyway

Reply:
Down the line we will either need to have cvmfs_hem_values in CL to pass values to cn_boot, or I suppose we could just use the CM values since there are no overlapping keys I think

Reply:
Re:moving before cert-manager, we can, although I believe the NFS is not ready by then usually so I don’t think it will speed up much. Even where it’s deployed now, it waits a bit for NFS provisioner to be ready. Down the line t might be helpful though so might as well flip it now

Reply:
Is the cache on the NFS? If so, we should probably bump it to more than 10G. We've had good experience in the past with 30G, although that may be a bit much if we're defaulting to 100G NFS disk so 20G?